src/main/java/io/mathdojo/useraccountservice/model/requestobjects/AccountRequest.java

```diff
@@ -26,12 +30,27 @@ public AccountRequest() {
      * 
      */
     public AccountRequest(boolean accountVerified, String name, String profileImageLink) {
-        this.accountVerified = accountVerified;
-        this.name = name;
-        this.profileImageLink = profileImageLink;
+        this(accountVerified, name, profileImageLink, null);
     }
-
-    public boolean isAccountVerified() {
+    /**
+     * Creates an instance of the AccountRequest class
+     * <p>
+     * 
+     * @param accountVerified  the status of the account being created
+     * @param name             the name of the prospective account holder
+     * @param profileImageLink an image of the profile user to be created
+     * @param id  the id of the account being created
+     * 
+     */
+    public AccountRequest(boolean accountVerified, String name,String profileImageLink, String id) {
+		super();
+		this.accountVerified = accountVerified;
+		this.name = name;
+		this.id = id;
+		this.profileImageLink = profileImageLink;
```

Comment:
Small question, do we have to have this `super()` call here? I would have thought it was `this()`.